### DISCUSSION TITLE
Mic Issues

### DISCUSSION THREAD

**Discussion by kboss805** [tesla-android/issue-tracker#293] [Q&A]
I have 2023.20.1 up and running. All is well except I can't seem to get the mic to work in either Bluetooth or box modes. Any other option I should check. Phone is connected to autobox and tesla.

**Comment by brswattt** [comment]
> I have 2023.20.1 up and running. All is well except I can't seem to get the mic to work in either Bluetooth or box modes. Any other option I should check. Phone is connected to autobox and tesla.

Did you ever figure this out?

**Comment by kboss805** [comment]
I did. I had to enable browser audio. It's a option in the tesla android "settings". Settings are accessible from main screen.

**Reply by brswattt** [reply]
I did, pretty sure. This is Carplay or AA? Do you think the Carplay/Android OS is blurry compared to the settings menu? That's my main gripe.
